### DISCUSSION TITLE
RFC: Fallback placeholder for ImageColumn

### DISCUSSION THREAD

**Discussion by pxlrbt** [filamentphp/filament#2344] [Polls]
Currently `ImageColumn`s height collapses when there is no image set. The default is to take the images size - so that behaviour is expected. But you can also set a size when using `->rounded()` or `->size()`. This results in rows with different height which looks inconsistent. 

![image](https://user-images.githubusercontent.com/22632550/167619029-ae936d8a-d2f3-42c6-92e4-0af28e91fde5.png)

I suggest to add an empty div with or without background as a fallback for the cases where the size is defined.

**Comment by danharrin** [comment]
Wanna do this @pxlrbt ?

**Reply by pxlrbt** [reply]
Yeah, I just revisited this as well 😅 I'll check whether this is done quickly.

**Reply by pxlrbt** [reply]
Done!
